docs/tools/new_tools_temp.md

```diff
@@ -46,23 +46,49 @@ Use pip to install:
     python -m pip install mbed-tools --pre
```

Comment:
Do we mention anywhere that `mbed-tools` is only compatible with python 3.6 and above?

Reply:
We say Python 3, not 3.6. Line 21

Reply:
OK. We should specify Python 3.6 or later, as it won't work correctly with Python 3.5 or below. Annoyingly Python 3 has many changes to standard library APIs between 3.0 and 3.6 that aren't entirely backwards compatible.